ocs_ci/utility/pagerduty.py

```diff
@@ -0,0 +1,332 @@
+import logging
+import os
+import requests
+import tempfile
+import time
+
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+from ocs_ci.utility.utils import exec_cmd
+
+logger = logging.getLogger(name=__file__)
+
+
+def set_pagerduty_integration_secret(integration_key):
+    """
+    Update ocs-converged-pagerduty secret. This is valid only on ODF Managed Service.
```

Comment:
I would note that we expect, that pagerduty integration for OCP alerting is already configured on ODF managed cluster, and that this function is just updating the pagerduty api key.

Reply:
Added note